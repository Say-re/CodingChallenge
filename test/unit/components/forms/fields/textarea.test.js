import React from 'react';
import { shallow } from 'enzyme';
import TextAreaComponent from '../../../../../src/components/forms/fields/textarea'

describe('Textarea Component tests', () => {
  test('Shallow TextareComponent Snapshot', async () => {
    const wrapper = shallow(<TextAreaComponent />)
    await expect(wrapper).toMatchSnapshot();
  })
})
