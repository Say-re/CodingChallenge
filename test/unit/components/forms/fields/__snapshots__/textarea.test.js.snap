// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Textarea Component tests Shallow TextareComponent Snapshot 1`] = `
<div>
  <p>
    Describe why you are a good fit for the job you are applying for
  </p>
  <textarea
    cols="50"
    id="shortAnswer"
    onChange={[Function]}
    placeholder="Testing"
    required={true}
    rows="10"
  />
</div>
`;
